### DISCUSSION TITLE
Add action button to the measure tool group

### DISCUSSION THREAD

**Discussion by salah-walid** [iTwin/itwinjs-core#3241] [Q&A] [answered]
Hi,

Is there a way to add a new action button to the measuring tool button group here ?
![image](https://user-images.githubusercontent.com/29980658/154706681-437f4de8-dcfd-47c2-91a2-a5cc13c5ac4b.png)

**Comment by aruniverse** [comment, marked as answer]
Unfortunately at the moment, there is no way to add a new button to an existing button group that you didn't create.
Looks like you're using itwin/web-viewer-react@^1.x. You can achieve the functionality you're looking for by, disabling the measure-tools we provide for you by default, and then re-construct the measure-tools button group yourself.
